zstacklib/zstacklib/utils/lvm.py

```
@@ -2817,16 +2817,23 @@ def report_config_changed():
-def subcmd(cmd, timeout=lvm_cmd_timeout_with_locking):
+def subcmd(cmd, timeout=lvm_cmd_timeout_with_locking, lockopts: list[str] | None = None):
     argv = [cmd]
+    lockopts = list(lockopts) if lockopts is not None else []
 
     if cmd in NOLOCK_CMDS:
         argv += ["--nolocking", "-t"]
-    elif cmd in TIMEOUT_CMDS:
-        argv = ["timeout", "-s", "SIGKILL", str(timeout)] + argv
+    else:
+        if cmd in TIMEOUT_CMDS:
+            argv = ["timeout", "-s", "SIGKILL", str(timeout)] + argv
```

Ruff (0.15.7)

[warning] Consider `["timeout", "-s", "SIGKILL", str(timeout), *argv]` instead of concatenation

Replace with `["timeout", "-s", "SIGKILL", str(timeout), *argv]`

(RUF005)